README.md

```diff
@@ -26,11 +26,10 @@ A JavaScript SDK for tracking events and revenue to [Amplitude](https://www.ampl
 * A [demo page](https://github.com/amplitude/GTM-Web-Demo) demonstrating a potential integration with Google Tag Manager.
 
 
-## React Native
-This library now supports react-native. It has two dependencies on react-native modules you will have to install yourself:
+## Deprecated - React Native
+As of > v7.4.4, this SDK no longer has support for react-native. We recently released a new SDK focused on providing a React Native and cross-platform first approach.  [Amplitude React Native SDK](https://github.com/amplitude/Amplitude-ReactNative)
```

Comment:
nit: any strong opinions between the last working build vs. the first breaking build (`>= 8.0.0`) ?

Reply:
yep, i like that wording better, added! @kelvin-lu